pyincore/analyses/joplincge/outputfunctions.py

```diff
@@ -67,5 +68,5 @@ def new_value(vars, soln, eqName, ittr):
 
 # Get differences function
 def get_diff(vars, soln, eqName, ittr):
-    difference = new_value(vars, soln, eqName, ittr) - base_value(vars, soln, eqName)
+    difference = new_value(vars, soln, eqName, ittr) - base_value(vars, eqName)
```

Comment:
@navarroc This seems like an important change. Can you give it a second look to make sure everything is ok?

Reply:
I don't see get_diff referenced anywhere in our code so it should be ok to fix this. Do you see it anywhere?

Reply:
In fact I don't see anything referencing outputfunctions.py. Seems like something we don't need/use. I'd leave it since it's there, but just mentioning.

Reply:
The function base_value only requires two parameters vars and eqName that is why I took out soln